<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="1" name="StateTest.TestPrefix-20151014155020" tests="5" time="0.086">
	<properties/>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
	<testcase classname="StateTest.TestPrefix" name="test_add_prefix" time="0.014"/>
	<testcase classname="StateTest.TestPrefix" name="test_initial_config" time="0.009"/>
	<testcase classname="StateTest.TestPrefix" name="test_move_prefix" time="0.026"/>
	<testcase classname="StateTest.TestPrefix" name="test_sensor_prefix_split" time="0.021"/>
	<testcase classname="StateTest.TestPrefix" name="test_del_prefix" time="0.015">
		<failure message="" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/home/saikappa/workspace/SciPass/python/t/StateTest.py", line 100, in test_del_prefix
    assert("10.0.21.0/24" not in prefixes)
AssertionError
]]>		</failure>
	</testcase>
</testsuite>
